src/index.js

```diff
@@ -72,7 +72,7 @@ client.on("message", (message) => {
 
 	// Sends typing game explanation to exercise channel
 	gameExplanation(message);
-	
+
```

Comment:
Not a big deal, but it looks like your editor is getting rid of whitespace on lines without any additional characters.
Perhaps it's a similar problem I had with my formatter?